.github/workflows/ci.yml

```diff
@@ -42,12 +42,7 @@ jobs:
       with:
         go-version: 1.19.x
         check-latest: true
-    - name: Cache
-      uses: actions/cache@v3
-      with:
-        path: ~/go/pkg/mod
-        key: ${{ runner.os }}-plugins-ci-${{ hashFiles('**/go.sum') }}
-        restore-keys: ${{ runner.os }}-plugins-ci
+        cache: true
```

Comment:
https://github.com/actions/setup-go#caching-dependency-files-and-build-outputs

Reply:
Oh neat, never seen this option. Curious, when was this added?